### PR TITLE
Building landing page

### PR DESCRIPTION
**Pull request by gagandeep95**
### What this PR does (required):
- This is the UI of the landing page.

### Screenshots / Videos (front-end only):
- (https://share.vidyard.com/watch/9U2VuvFsBx5dwEciWfYJNZ?)

### Any information needed to test this feature (required):
- Please  use this command - npm install before testing this feature as I have added a couple of packages in to order to use date picker feature
### Any issues with the current functionality (optional):
- Although I have added the conditional styles to the navbar, they are working correctly when the page load or refresh, However, When clicking on the Link Button, the component does not re-render.
- It might conflict with other branches as i modified Navbar.tsx, theme.ts, LoggedInBar.tsx files.
- Please watch the above-attached video to understand how the component does not re-render and does not change the style when clicking on the Link button.

### CODE REVIEW

File: client/src/components/Home/FindDogSitter.tsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+            value={value}
+            onChange={(newValue) => {
+              setValue(newValue);
+            }}
+            renderInput={(params) => <TextField variant="outlined" {...params} />}
+          />
+        </LocalizationProvider>
+      </Box>
+      <Button color="primary" size="large" variant="contained">
+        FIND MY DOG SITTER
```
Comment (ezeilo-su) [inline client/src/components/Home/FindDogSitter.tsx:39]: Please avoid using CAPITALCASE for text, some screen readers interpret this text as abbreviations, you can resort to using a CSS class to transform text instead.

File: client/src/themes/theme.ts [hunk 2]
```diff
@@ -33,4 +41,14 @@ export const theme = createMuiTheme({
   shape: {
     borderRadius: 5,
   },
+  spacing: 2, //setting pixel size of 2
```
Comment (ezeilo-su) [inline client/src/themes/theme.ts:44]: Let's get rid of this comment.

**Review by ezeilo-su — CHANGES_REQUESTED** [review]
UI looks good! A few minor comments, though. :+1:

File: client/src/pages/Home/Homepage.tsx [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  const classes = useStyles();
+
+  return (
+    <Grid container>
+      <Grid item sm={12} md={6} className={classes.findDogSitter} direction="column">
+        <Typography variant="h2">Find the care your dog deserves</Typography>
+        <FindDogSitter />
+      </Grid>
+      <Grid item sm={12} md={6} className={classes.banner}>
+        <img src={banner} className={classes.image} />
```
Comment (rajivtitus) [inline client/src/pages/Home/Homepage.tsx:16]: It is a good practice to have 'alt' attributes for HTML image elements

File: client/src/themes/theme.ts [hunk 4]
```diff
@@ -33,4 +41,14 @@ export const theme = createMuiTheme({
   shape: {
     borderRadius: 5,
   },
+  spacing: 2,
+  breakpoints: {
+    values: {
+      xs: 0,
```
Comment (rajivtitus) [inline client/src/themes/theme.ts:47]: 💯

File: client/src/components/NavBar/AuthBars/LoggedInBar.tsx [hunk 5]
```diff
@@ -4,30 +4,40 @@ import Typography from '@material-ui/core/Typography';
...
 const LoggedInBar = (): JSX.Element => {
   const classes = useStyles();
+  const { pathname } = useLocation();
 
   return (
     <Grid container className={classes.navButtons}>
       <Grid item>
         <Button component={Link} to="/notifications" color="secondary" size="large" variant="text">
-          <Typography variant="h3">Notifications</Typography>
+          <Typography className={pathname === '/' ? classes.homeTypography : classes.typography} variant="h3">
```
Comment (rajivtitus) [inline client/src/components/NavBar/AuthBars/LoggedInBar.tsx:17]: Optional: You can use a lightweight package list [clsx](https://www.npmjs.com/package/clsx) to conditionally apply class names - makes things a lot cleaner

File: client/src/components/Home/FindDogSitter.tsx [hunk 6]
```diff
@@ -0,0 +1,47 @@
+import { useState } from 'react';
+import Box from '@material-ui/core/Box';
+import TextField from '@material-ui/core/TextField';
+import Button from '@material-ui/core/Button';
+import Stack from '@mui/material/Stack';
+import AdapterDateFns from '@mui/lab/AdapterDateFns';
+import LocalizationProvider from '@mui/lab/LocalizationProvider';
+import DatePicker from '@mui/lab/DatePicker';
+import { Typography } from '@material-ui/core';
```
Comment (rajivtitus) [inline client/src/components/Home/FindDogSitter.tsx:9]: If you are importing the Typography as a named import, might as well import the above components within this import statement :)

File: client/src/components/Home/useStyles.ts [hunk 7]
```diff
@@ -0,0 +1,5 @@
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles(() => ({ typography: { textTransform: 'uppercase' } }));
```
Comment (rajivtitus) [inline client/src/components/Home/useStyles.ts:3]: 👍🏽

**Review by rajivtitus — APPROVED** [review]
Good work! Just a couple of minor comments to address before merging 😄

File: client/src/pages/Home/Homepage.tsx (hunk 3 above)
Comment (gagandeep95) [reply client/src/pages/Home/Homepage.tsx:16]: Thanks Rajiv, I will make it correct

**Pull request merged**